### DISCUSSION TITLE
Better BulletPoint handling in PowerPoint

### DISCUSSION THREAD

**Discussion by klein-artur** [OfficeDev/office-js#5065] [Ideas]
Using the OfficeJS api in PowerPoint, you can get the bulletFormat like this:

```typescript
textRange.paragraphFormat.bulletFormat
```

But the only thing you can get and set is the visibility. But we need way more, like the type of  bullet point per level, maybe the character, etc. etc.

Is there a plan to introduce such methods?

**Comment by ElizabethSamuel-MSFT** [comment]
Related issue: https://github.com/officedev/office-js/issues/5368
